backend/src/routes/middleware/expressLogger.ts

```diff
@@ -35,13 +29,11 @@ const morganLog = promisify(
 )
 
 export async function expressLogger(req: Request, res: Response, next: NextFunction) {
-  req.reqId = (req.headers['x-request-id'] as string) || uuidv4()
   req.log = log.child({
-    id: req.reqId,
-    user: req.user?.id,
```

Comment:
are we losing user from the request logs?

Reply:
These lines did nothing because the middleware that added the user to the request obtain had not been run yet.